NeuroID/src/main/java/com/neuroid/tracker/utils/NIDVersion.kt

```diff
@@ -1,10 +1,19 @@
 package com.neuroid.tracker.utils
 
 import com.neuroid.tracker.BuildConfig
+import com.neuroid.tracker.NeuroID
 
 object NIDVersion {
     fun getSDKVersion(): String {
-        return "5.android-" + BuildConfig.VERSION_NAME
+        val rnText = NeuroID.getInstance()?.isRN?.let {
```

Comment:
consider: 

var rnText = ""
rnText = NeuroID.getInstance()?.isRN?.let {....

just in case NeuroID.getInstance() returns null

Reply:
I tried to put that and it failed because the `NeuroID.getInstance()?.isRN?.let` returns an optional string and `var rnText` is a non-optional string